packages/shared/src/components/search/SearchBarSuggestionList.tsx

```diff
@@ -63,13 +63,13 @@ export function SearchBarSuggestionList({
           tag="a"
           origin={origin}
           suggestion={suggestion}
-          key={suggestion.prompt}
+          key={suggestion.question}
           href={getSearchUrl({
             id: suggestion.id,
-            question: suggestion.prompt,
```

Comment:
Somehow we need to ensure we distinguish between. prompt and question else it will become very unclear.
Not really sure what's right here, but maybe up for discussion.

Reply:
We could consolidate these so they use the same attribute name?

Reply:
`question` is column from our API and `prompt` coming from Magni's session. Quickest just to rename it on our client or long term make it consistent across our services.